bokeh/colors/color.py

```diff
@@ -308,16 +308,20 @@ def to_css(self) -> str:
             return f"rgba({self.r}, {self.g}, {self.b}, {self.a})"
 
     def to_hex(self) -> str:
-        ''' Return a hex color string for this RGB color.
+        ''' Return a hex color string for this RGB(A) color.
 
-        Any alpha value on this color is discarded, only hex color strings for
-        the RGB components are returned.
+        Any alpha value if only included in the output string if it is less
```

Comment:
*is* only included?

Reply:
Oops! I've snuck the correction into the contouring docs PR.